config/navigation.yaml

```diff
@@ -1,7 +1,12 @@
 drive:
-  max_driving_effort: 1
-  min_driving_effort: -1
-  turning_p: 10.0
+  map:
+    max_driving_effort: 1
```

Comment:
Not sure if you want to change this, since it's probably insignificant, but I noticed that turning_p is a float while the driving efforts are ints. But then in drive.py, line 11, you pass in floats for all of the default values. So the 1.0 and -1.0 in drive.py could be changed to match this file.

Reply:
I'll change them to floats in the yaml as well